app/v2/models/request/UpscanInitiate.scala

```diff
@@ -24,8 +24,5 @@ object UpscanInitiate {
 
 case class UpscanInitiate(
   callbackUrl: String,
-  successRedirect: Option[String],
```

Comment:
maybe we need these few of these optional fields later - if required we can later as well

Reply:
Yeah we might choose to use them later on, but since they are are always being set to None at the moment and are optional, I thought it was best to remove something we don't yet need